course/templates/course/home.html

```diff
@@ -2,16 +2,18 @@
 {% load i18n %}
 
 {% block content %}
+  {% block cover %}
   <div class="jumbotron">
-    {% trans "RELATE" as RELATE %}
-    <h1>{% blocktrans %} Welcome to {{ RELATE }} {% endblocktrans %}</h1>
+    <h1>{% blocktrans %} Welcome to {{ relate_site_name }} {% endblocktrans %}</h1>
```

Comment:
Shouldn't that use a `{% with` block to feed in the site name? Otherwise all the translation literals need to be changed.

Reply:
You're right. How come I didn't think of that...

Reply:
Updated.